src/js/components/molecules/tabs/tabs.ts

```diff
@@ -29,23 +29,21 @@ export class Tabs extends Vue implements TabsProps {
     setActiveTab() {
         const tabs = this.$refs.tab;
 
-        if (!tabs) { return; }
+        if (!tabs) return;
 
         tabs[this.activeTab].scrollIntoView({ behavior: 'smooth', inline: 'center', block: 'nearest' });
 
         setTimeout(() => this.positionIndicator(), 50);
     }
 
-    getLinkClassList(index) {
-        return {
-            'tabs__list_link--active': index === this.activeTab,
-        };
-    }
+    getLinkClassList = index => ({
+        'tabs__list_link--active': index === this.activeTab,
+    });
 
-    async positionIndicator() {
+    positionIndicator() {
         const tabs = this.$refs.tab;
 
-        if (!tabs) { return; }
+        if (!tabs) return;
```

Comment:
avoid these single line if statements. @see https://www.crockford.com/code.html#if%20statement

```suggestion
        if (!tabs) {
            return;
        }
```